index.yaml

```diff
@@ -97,30 +215,4 @@ entries:
     urls:
     - https://github.com/scalar-labs/helm-charts/releases/download/schema-loading-1.3.0/schema-loading-1.3.0.tgz
     version: 1.3.0
-  envoy:
```

Comment:
The order has been switched from above. (This is because it was generated automatically.)